### PR TITLE
fozzie-components@4.2.0 - icing phase 2 (part 2 of 3)

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- Updated version of `f-button`.
- New colour scheme and border radius from `pie-design-tokens` in line with icing phase 2.

### Changed
- New colour scheme and border radius from `pie-design-tokens` in line with icing phase 2.

### Changed
- Updated version of `f-button`.
- New border radius from `pie-design-tokens` in line with icing phase 2.
- Close button type from `ghostTertiary` to `inverse`.

### Changed
- New colour tokens from `pie-design-tokens` in line with icing phase 2.

### Changed
- New colour tokens from `pie-design-tokens` in line with icing phase 2.

### Changed
- Updated version of `f-button` and `f-mega-modal`.
!Note: Because this component is not in prod yet, more changes for icing phase 2 will come later.

### CODE REVIEW

File: packages/components/molecules/f-mega-modal/src/components/MegaModal.vue [hunk 1]
```diff
@@ -305,7 +305,7 @@ export default {
 
 .c-megaModal-content {
     background-color: $color-container-default;
-    border-radius: $border-radius;
+    border-radius: $radius-rounded-d;
```
Comment (DamianMullins) [inline packages/components/molecules/f-mega-modal/src/components/MegaModal.vue:308]: Should this be `$radius-rounded-c`?

Comment (dandel10n) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue:308]: In line with PIE spreadsheets (can't attach the link as this is a public repo) modals should have `$radius-rounded-d` radius. If you see a different value on some designs, it is better to ask the designers as there are a lot of misalignments in the designs as it is all still WIP and some values don't match.

Comment (DamianMullins) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue:308]: Ignore, `$radius-rounded-d` is correct.

**Review by AlixRichards — DISMISSED** [review]

**Review by DamianMullins — DISMISSED** [review]

**Review by SachaZvetelman — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

File: packages/components/molecules/f-mega-modal/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 
 
+v3.0.0
+------------------------------
+*October 5, 2021*
+
+### Changed
+- Updated version of `f-button`.
+- New border radius from `pie-design-tokens` in line with icing phase 2.
+- Close button type from `ghostTertiary` to `iverse`.
```
Comment (ashleynolan) [inline packages/components/molecules/f-mega-modal/CHANGELOG.md:14]: Super NIT  should be `inverse`

Comment (dandel10n) [reply packages/components/molecules/f-mega-modal/CHANGELOG.md:14]: Oops

**Pull request merged**